pkg/controller_helper/model_source/uri.go

```diff
@@ -26,18 +26,23 @@ import (
 var _ ModelSourceProvider = &URIProvider{}
 
 const (
-	OSS = "OSS"
+	OSS    = "OSS"
+	OLLAMA = "OLLAMA"
 )
 
 type URIProvider struct {
-	modelName string
-	protocol  string
-	bucket    string
-	endpoint  string
-	modelPath string
+	modelName    string
+	protocol     string
+	bucket       string
```

Comment:
We may need to refactor this part in the future, divide the URIProvider to more specified ones, but not a hurry.